test/controllers/sessions_controller_test.rb

```diff
@@ -1,5 +1,40 @@
 require "test_helper"
 
 describe SessionsController do
+  def create
```

Comment:
Why do you have methods here?  This is the **test** file.  You should have `it` blocks and expectations.  

I do not understand what's going on here.